packages/target-chrome-core/src/create-chrome-target.js

```diff
@@ -161,6 +162,7 @@ function createChromeTarget(
         debug('Disabling animations');
         await evaluateOnNewDocument(`(${disableAnimations})(window);`);
       }
+      await evaluateOnNewDocument(`(${disablePointerEvents})(window);`);
```

Comment:
Is there a use case where people would like to "enable pointer-events" just like the `chromeEnableAnimations` option above?

If so, there should be a way (e.g. flag from `package.json` or command line) to enable/disable it 👍

Reply:
I honestly don't think so. Disabling animations is much more invasive and could potentially break apps, this shouldn't.